tasks/databases.yml

```diff
@@ -5,4 +5,5 @@
     collation: "{{ item.collation | default('utf8_general_ci') }}"
     encoding: "{{ item.encoding | default('utf8') }}"
     state: present
+  sudo: no
```

Comment:
I'd rather not set `sudo: no` for all these tasks, but be more explicit about which (if any) tasks truly _must_ be run as sudo (e.g. the `apt` or `yum` tasks). That way, for the minority of playbooks which are fully run without `sudo: yes`, the adding `sudo: no` is redundant.